src/pages/Booking/components/Movie/BookingMovie.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import MovieLists from './components/MovieLists.js';
+import MoviePosts from './components/MoviePosts.js';
+import styled from 'styled-components';
+
+function BookingMovie({ bookingData, handleSelectMovie, selectMovie }) {
+  return (
+    <BookingContainer>
+      <BookingFlexBox>
+        <H2>영화</H2>
+        <MovieAll>전체</MovieAll>
+        <MovieList>
+          {bookingData &&
+            bookingData.map(movies => (
```

Comment:
옵셔널 체이닝(optional chaining)으로 다음과 같이 동일하게 구현할 수 있습니다.
많이 사용되는 문법이니 꼭 찾아보고 적용해주세요!
```suggestion
          {bookingData?.map(movies => (
```